CHANGELOG.md

```diff
@@ -6,6 +6,9 @@
 - Bundle BitBox02 v9.5.0 firmware
 - Run BitBoxApp installer as admin by default on Windows
 - Close a running BitBoxApp instance on Windows when installing an update to ensure success
+- Show connection error in detail
+- Connect default BTC/LTC full nodes on port 443 to work around firewalls blocking the 5XXXXX ports
```

Comment:
One off too many Xs :) A port like 512345 is invalid.

Reply:
Damn, again I merged based on the approve checkmark :O Fixed in a new PR https://github.com/digitalbitbox/bitbox-wallet-app/pull/1224